docs/peer-dependencies.md

```diff
@@ -1,14 +0,0 @@
-# Peer Dependencies
```

Comment:
this file was no longer referenced and all the docs tell people to read their warnings from `npm` or `yarn`